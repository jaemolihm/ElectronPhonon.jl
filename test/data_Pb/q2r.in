&input
  fildyn = 'dyn_dir/pb.dyn'
  zasr   = 'crystal'
  flfrc  = 'pb.fc'
/

