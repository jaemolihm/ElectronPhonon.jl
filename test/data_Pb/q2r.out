
     Program Q2R v.6.7MaX starts on  9Sep2021 at 18:45:19 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes

     reading grid info from file dyn_dir/pb.dyn0    
  reading force constants from file dyn_dir/pb.dyn1
  Dielectric Tensor not found
  nqs=            1
 q=   0.00000000  0.00000000  0.00000000
  reading force constants from file dyn_dir/pb.dyn2
  nqs=            8
 q=  -0.33333333  0.33333333 -0.33333333
 q=   0.33333333  0.33333333 -0.33333333
 q=   0.33333333  0.33333333  0.33333333
 q=  -0.33333333  0.33333333  0.33333333
 q=   0.33333333 -0.33333333  0.33333333
 q=  -0.33333333 -0.33333333 -0.33333333
 q=   0.33333333 -0.33333333 -0.33333333
 q=  -0.33333333 -0.33333333  0.33333333
  reading force constants from file dyn_dir/pb.dyn3
  nqs=            6
 q=   0.00000000  0.66666667  0.00000000
 q=  -0.66666667  0.00000000  0.00000000
 q=   0.00000000 -0.66666667  0.00000000
 q=   0.00000000  0.00000000  0.66666667
 q=   0.00000000  0.00000000 -0.66666667
 q=   0.66666667  0.00000000  0.00000000
  reading force constants from file dyn_dir/pb.dyn4
  nqs=           12
 q=   0.66666667  0.00000000  0.66666667
 q=   0.66666667  0.00000000 -0.66666667
 q=  -0.66666667  0.00000000 -0.66666667
 q=  -0.66666667  0.00000000  0.66666667
 q=   0.00000000 -0.66666667  0.66666667
 q=  -0.66666667  0.66666667  0.00000000
 q=  -0.66666667 -0.66666667  0.00000000
 q=   0.66666667  0.66666667  0.00000000
 q=   0.66666667 -0.66666667  0.00000000
 q=   0.00000000  0.66666667  0.66666667
 q=   0.00000000  0.66666667 -0.66666667
 q=   0.00000000 -0.66666667 -0.66666667

      q-space grid ok, #points =   27

      fft-check success (sum of imaginary terms < 10^-12)
 
     Q2R          :      0.01s CPU      0.04s WALL

 
   This run was terminated on:  18:45:19   9Sep2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
